### PR TITLE
Add pt-BR translation

### PR DESCRIPTION
**Pull request by danilolmoura**
- Add pt-BR translation
- Change version to v0.9.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fusliez-notes",
-  "version": "0.9.0",
+  "version": "0.9.1",
```
Comment (Kedyn) [inline package.json:3]: Would like to add more things for this patch, so would you mind reverting this change.

**Review by Kedyn — COMMENTED** [review]
Awesome work, just minor problem with versioning. The versions should be set after all changes are completed by the maintainers.

File: package.json (hunk 1 above)
Comment (danilolmoura) [reply package.json:3]: That's ok

**Comment by danilolmoura** [comment]
@Kedyn, version reverted in [this commit](https://github.com/Kedyn/fusliez-notes/pull/55/commits/ab1b969cfe3304c1a81fe265c159544d418bd2ce)

**Comment by Kedyn** [comment]
Awesome, thank you! I will try to publish this change by today.

**Pull request merged**